default/methods/task/stats/fast_stats.py

```diff
@@ -0,0 +1,55 @@
+from methods.regular.regular_api import *
+from shared.database.task.job.job import Job
+from shared.database.user import User
+from shared.database.task.task import Task
+from shared.database.annotation.instance import Instance
+
+@routes.route('/api/job/<job_id>/stat', methods=["GET"])
+def job_stat(job_id):
+    with sessionMaker.session_scope() as session:
+        job = Job.get_by_id(session, job_id).__dict__
+        job.pop('_sa_instance_state', None)
+        return jsonify({
+            "total": job["stat_count_tasks"],
+            "completed": job["stat_count_complete"]
+            })
+
+@routes.route('/api/job/<job_id>/user/<user_id>/stats', methods=["GET"])
+def jon_user_stats(job_id, user_id):
+    with sessionMaker.session_scope() as session:
+        tasks = Task.list(
+            session, 
+            job_id=job_id,
+        )
+        instances_of_tasks = []
+        task_list = []
+        for task in tasks:
+            task_dict = task.__dict__
+            task_dict.pop('_sa_instance_state', None)
+            task_dict.pop('job', None)
+            task_dict.pop('incoming_directory', None)
+            task_list.append(task_dict)
+
+        return_task_list = [task for task in task_list if task["assignee_user_id"] == int(user_id)]
```

Comment:
Good job figuring out the API endpoints Vitalii! Just for your information, future direction of this would be to integrate it with the report runner system (Instead of using task.list()) but this is great for the v1 of it. We can optimize the queries in the future.

Reply:
Yeah, I tried with report runner but at this point it required report template